msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -12,15 +12,35 @@ The resource-specific consent (RSC) permissions model, originally developed for
 
 ## Enable bots to receive all channel or chat messages
 
-> [!NOTE]
->
-> The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md).
-
 The `ChannelMessage.Read.Group` and `ChatMessage.Read.Chat` RSC permissions are being extended to bots. With user consent and app installation, these permissions:
 
 * Allow a specified graph application to get all messages in channels and chats, respectively.
 * Enable a bot defined in the app manifest to receive all conversations messages without being @mentioned in relevant contexts where the permissions apply.
 
+> [!NOTE]
+>
+> * The ability for bots to receive all messages in chats using `ChatMessage.Read.Chat` is available only in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md) and will only be enabled after a new installation into a chat. 
+> * Once enabled, the bot will continue to receive all messages even when the client switches out of public developer preview. 
+> * If you have an app that is currently using the `ChatMessage.Read.Chat` RSC permission for Graph scenarios, then you should test the app following the guide below and modify the app if needed.
+
+```csharp
```

Comment:
Can we move this code sample above the [NOTE] section?

Reply:
Moved the note section.